examples/styling/image-external-no-color.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
```

Comment:
Do we need a test for this? I mean, the issue seems to be a bug. If there the example is really useful for the users (check with @makella ) we can add it. If the example is only useful for ourselves (cvl devs) we could add it to `debug/`, although we have a test

Reply:
👍 That's right, it is a bug, and nothing really special in the example itself for the users. 

I'm gonna move it to `debug/` (I didn't know about that possibility)

Reply:
the interactive-based style that we have in the dev center example is this one:
https://carto.com/developers/carto-vl/examples/#example-interactive-based-styling

for images, we don't have an example that shows style changes on hover/click so I think we are ok.

if there is something special that should be pointed out about the workflow for interactive based styling with images, we can definitely add.

Reply:
Hi @makella ! This example helped us debugging the error, but it works just as the example you have posted, so I moved it to the "internal" debug folder. Is it fine?

Reply:
yep!! thank you!!